test/reactors/V2DutchOrderReactor.t.sol

```diff
@@ -0,0 +1,293 @@
+            outputOverrides: ArrayBuilder.fill(2, 1.1 ether)
+        });
+        V2DutchOrder memory order = V2DutchOrder({
+            info: OrderInfoBuilder.init(address(reactor)).withSwapper(swapper),
+            cosigner: vm.addr(cosignerPrivateKey),
+            input: DutchInput(tokenIn, 1 ether, 1 ether),
+            outputs: OutputsBuilder.singleDutch(address(tokenOut), 1 ether, 0.8 ether, swapper),
+            cosignerData: cosignerData,
+            cosignature: bytes("")
+        });
+        order.cosignature = cosignOrder(order.hash(), cosignerData);
+        SignedOrder memory signedOrder =
+            SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
+        vm.expectRevert(V2DutchOrderReactor.InvalidOutputOverride.selector);
+        fillContract.execute(signedOrder);
+    }
+
+    function testOverrideInput() public {
+        uint256 outputAmount = 1 ether;
+        uint256 overriddenInputAmount = 0.7 ether;
+        tokenIn.mint(swapper, overriddenInputAmount);
+        tokenOut.mint(address(fillContract), outputAmount);
+        tokenIn.forceApprove(swapper, address(permit2), type(uint256).max);
+        CosignerData memory cosignerData = CosignerData({
+            decayStartTime: block.timestamp,
+            decayEndTime: block.timestamp + 100,
+            exclusiveFiller: address(0),
+            inputOverride: overriddenInputAmount,
+            outputOverrides: ArrayBuilder.fill(1, 1 ether)
+        });
+
+        V2DutchOrder memory order = V2DutchOrder({
+            info: OrderInfoBuilder.init(address(reactor)).withSwapper(swapper),
+            cosigner: vm.addr(cosignerPrivateKey),
+            input: DutchInput(tokenIn, 0.8 ether, 1 ether),
+            outputs: OutputsBuilder.singleDutch(address(tokenOut), outputAmount, outputAmount, swapper),
+            cosignerData: cosignerData,
+            cosignature: bytes("")
+        });
+        order.cosignature = cosignOrder(order.hash(), cosignerData);
+        SignedOrder memory signedOrder =
+            SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
+        fillContract.execute(signedOrder);
+        assertEq(tokenIn.balanceOf(swapper), 0);
+        assertEq(tokenOut.balanceOf(swapper), outputAmount);
+        assertEq(tokenIn.balanceOf(address(fillContract)), overriddenInputAmount);
+    }
+
+    function testOverrideOutput() public {
+        uint256 overriddenOutputAmount = 1.1 ether;
+        uint256 inputAmount = 1 ether;
+        tokenIn.mint(swapper, inputAmount);
+        tokenOut.mint(address(fillContract), overriddenOutputAmount);
+        tokenIn.forceApprove(swapper, address(permit2), type(uint256).max);
+        CosignerData memory cosignerData = CosignerData({
+            decayStartTime: block.timestamp,
+            decayEndTime: block.timestamp + 100,
+            exclusiveFiller: address(0),
+            inputOverride: inputAmount,
```

Comment:
if youre not overriding the input for this test, shouldnt this just be 0?

Reply:
true, fixed